docs/04-guides/add-viewership/02-api.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+pagination_next: guides/playback-a-video-stream
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them ready
+for optimized playback. Videos can be provided either by you (static content) or
+your users, given your application offers an interface for them to do so.
+
+## Step 1: Configure your player
+
+To display viewership data, you need to configure your player first.
+We recommend using the [Livepeer player](https://livepeerjs.org/) as the player
+is already configured. You can follow this [guide](guides/playback-a-video-stream) to
+setup the Livepeer player.
+
+## Step 2: Get the `playbackId` of an existing asset
+
+Get the `playbackId` of an existing asset. An asset's `playbackId` can be found by making 
+an API request using a `stream.id` or `asset.id`. You can find the `playbackId` in the response object of the 
+[asset](https://docs.livepeer.studio/reference/api/get-assets#response-example) request.
+
+If you haven't created an asset yet, follow the 
+[upload a video asset guide](/guides/on-demand/upload-video-asset#api) to do so.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="JavaScript" label="JavaScript">
+
+```js
+const getPlaybackId = await fetch("https://livepeer.studio/api/asset/{id}", {
```

Comment:
@ichung08 same feedback as the other guide. Let's be more conscientious about this variable names and code snippets.